ci/openshift.yml

```diff
@@ -50,9 +50,11 @@ env:
   # ⬇️ EDIT if you wish to set the kube context's namespace after login. Leave blank to use the default namespace.
   OPENSHIFT_NAMESPACE: ""
 
-  # If you wish to manually provide the APP_NAME and TAG, set them here, otherwise they will be auto-detected.
+  # If you wish to manually provide the APP_NAME and TAGS, set them here, otherwise they will be auto-detected.
```

Comment:
since new-app only works for one tag, i think this should still be singular. any tag after the first won't work

Reply:
Okay, makes sense.